src/components/additem/useAddItemSharedData.js

```diff
@@ -0,0 +1,22 @@
+import { useState } from "react";
+
+export const useAddItemSharedData = () => {
```

Comment:
P2:
data보다 state가 더 적절한 이름인 것 같습니다.

```suggestion
export const useAddItemState= () => {
```